models/inventory/StockTransfer.ts

```diff
@@ -246,6 +270,30 @@ export abstract class StockTransfer extends Transfer {
     await invoice.setAndSync('stockNotTransferred', notTransferred);
   }
 
+  async _updateItemsReturned() {
+    if (!this.returnAgainst) {
+      return null;
+    }
+
+    const returnDocs = await this.fyo.db.getAll(this.schema.name, {
+      filters: {
+        returnAgainst: this.returnAgainst,
+        submitted: true,
+        cancelled: false,
+      },
+    });
+
+    const isItemsReturned = returnDocs.length;
+
+    const referenceDoc = await this.fyo.doc.getDoc(
```

Comment:
You don't need to use use `getAll` then `getDoc` to get the return against doc. You can just use `this.loadAndGetLink('returnAgainst')` instead.

Reply:
corrected